src/library_gl.js

```diff
@@ -1400,6 +1411,97 @@ var LibraryGL = {
     GLctx.bufferSubData(target, offset, HEAPU8.subarray(data, data+size));
   },
 
+  // Queries EXT
```

Comment:
Thinking about how to structure this, it will be silly if one needs to create two different types of query objects, "EXT query objects" and "WebGL 2 query objects" if one wants to do both types of queries in an application.

Let's pause here for a second, I'll raise this up to the WebGL spec mailing list and we'll see if there can be some consensus.